docs/developing/wallet-setup.md

```diff
@@ -12,11 +12,13 @@ Wallets allow you to interact with dApps without exposing your account's private
 
 Which wallet you install is up to you and whether you want to install a wallet on your computer, in a browser extension, or as a mobile app.
 
-Mobile apps include [Temple](https://templewallet.com/), [Kukai](https://wallet.kukai.app/), and [Umami](https://umamiwallet.com/).
+These are some wallets that you can use with Tezos:
 
-Browser extensions include the [Temple](https://templewallet.com/) browser extension.
-
-Desktop wallets for Tezos include [Kukai](https://wallet.kukai.app/) and [Umami](https://umamiwallet.com/).
+- [Temple](https://templewallet.com/) mobile application and browser extension
+- [Umami](https://umamiwallet.com/) standalone application for Windows, MacOS, and Linux
+- [Kukai](https://wallet.kukai.app/) federated identity application, which uses accounts linked to other accounts such as email or social media accounts
+- [Trust](https://trustwallet.com/tezos-wallet) mobile application and browser extension
+- [Ledger](https://www.ledger.com/) hardware wallets
```

Comment:
Also, we might want to list Airgap as well? https://airgap.it/#download

They have a rather unique approach for a cold/hot wallet.

Reply:
I don't want to get into airgap vault/wallet here but I added it to the list.